transport/network/src/network.rs

```diff
@@ -308,7 +308,7 @@ where
     pub async fn find_peers_to_ping(&self, threshold: SystemTime) -> crate::errors::Result<Vec<PeerId>> {
         let stream = self
             .db
-            .get_network_peers(PeerSelector::default().with_last_seen_lte(threshold), true)
+            .get_network_peers(PeerSelector::default().with_last_seen_lte(threshold), false)
```

Comment:
This function signature would be better readable if it takes the order enum instead of a boolean:

e.g.: `Order::Asc` or `Order::Desc`

Reply:
True, but the Order is an SQL implementation utility and would leak the entire implementation to an otherwise agnostic network crate.